packages/SwingSet/src/liveslots/vatstore-usage.md

```diff
@@ -63,50 +62,53 @@ In a c-list or virtualized data, you may see vrefs like these:
+  * If the Kind is multi-facet, the value will be a string with one letter for each facet, in the same order as their Facet ID. `n` is used to indicate neither reachable nor recognizable. For example, a value of `rsnr` means there are four facets, the first (`o+11/1:0`) and last (`o+11/1:3`) are reachable, the second (`o+11/1:1`) is recognizable, and the third (`o+11/1:2`) is neither.
 
 
 # Durable Kinds
 
-Virtual objects are held on disk, not RAM, which makes them suitable for high-cardinality data: many objects, most of which are "idle" at any given time. However virtual objects do not survive a vat upgrade. For this, vats should define one or more "Durable Kinds" instead.
+Virtual objects are held on disk, which makes them suitable for high-cardinality data that may not fit in RAM: many objects, most of which are "idle" at any given time. However virtual objects do not survive a vat upgrade. For this, vats should define one or more "Durable Kinds" instead.
 
 Durable Kinds are defined just like virtual Kinds, but they use a different constructor (`defineDurableKind` instead of `defineKind`), which requires a "handle" created by `makeKindHandle`. Durable virtual objects can only hold durable data in their `state`.
 
-The KindHandle is a durable virtual object of a special internal Kind. This is the first Kind allocated, so it generally gets Kind ID "1", and the handles will get vrefs of `o+1/N`.
+The KindHandle is a durable virtual object of a special internal Kind. This is the first Kind allocated, so usually it gets Kind ID `1` and the handles get vrefs like `o+1/${kindHandleID}`.
 
 
 # Kind Metadata
 
-For each virtual object kind that is defined we store a metadata record for purposes of scanning directly through the defined kinds when a vat is stopped or upgraded.  For durable kinds this record is stored in `vom.dkind.${kindID}`; for non-durable kinds it is stored in `vom.vkind.${kindID}`.  Currently this metadata takes the form of a JSON-serialized record `{ kindID, tag }`, where the `kindID` property is the kind ID (redundantly) and `tag` is the tag string as provided in the `defineKind` or `makeKindHandle` call.
-
-Durable kinds need to store their `nextInstanceID` in the DB, so that subsequent versions can begin allocating new instances from a non-overlapping starting point. For durable kinds, the metadata record is initially `{ kindID, tag, nextInstanceID }`, and the `nextInstance` property is updated after every durable-object allocation.
+For each virtual object kind that is defined, we store a metadata record for purposes of scanning directly through the defined kinds when a vat is stopped or upgraded. For durable kinds this record is stored in `vom.dkind.${kindID}`; for non-durable kinds it is stored in `vom.vkind.${kindID}`. Currently this metadata takes the form of a JSON-serialized record with the following properties:
+* `kindID`, the kind ID (redundantly with the storage key)
```

Comment:
What is the format of a value for the Kind metadata record `kindID` field?
* integer, e.g. `10`
* numeric string, e.g. `"10"`
* prefixed string, e.g. `"o+10"`